### PR TITLE
Add guildicon command

### LINKED ISSUE #333: Server icon command

### PR DESCRIPTION
**Pull request by beescuit**
- Closes #333

### CODE REVIEW

**Comment by bolsomito** [comment]
falou tá falado, por mim aprova samerda

File: src/locales/en-US/commands.json [hunk 1]
```diff
@@ -410,5 +410,9 @@
     "noText": "You need to give me some text",
     "tooLongText": "Your text is longer than 150 characters",
     "changedSuccessfully": "Personal text changed to {{profileText}}!"
+  },
+  "guildicon": {
```
Comment (bolsomito) [inline src/locales/en-US/commands.json]: nem por isso, tá falando commandDescription e commandUsage (acho q o último não é preciso botar né)

Comment (bolsomito) [reply src/locales/en-US/commands.json]: faltando*

File: src/commands/misc/guildicon.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const { CommandStructures, SwitchbladeEmbed } = require('../../')
+const { Command, CommandRequirements } = CommandStructures
+
+module.exports = class GuildIcon extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'guildicon'
```
Comment (Doges) [inline src/commands/misc/guildicon.js]: Guild is for API, not interface.

File: src/commands/misc/guildicon.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  }
+
+  run ({ t, author, channel, guild }) {
+    const embed = new SwitchbladeEmbed(author)
+    channel.startTyping()
+    if (guild.iconURL) {
+      embed.setImage(guild.iconURL)
+        .setDescription(t('commands:guildicon.iconDescription', {guild: guild.name}))
+    } else {
+      embed.setDescription(t('commands:guildicon.noIcon'))
```
Comment (Doges) [inline src/commands/misc/guildicon.js]: .setColor(Constants.ERROR_COLOR) :wesmart:

File: src/commands/misc/guildicon.js (hunk 2 above)
Comment (Doges) [reply src/commands/misc/guildicon.js]: f it, keep it as guildinfo

**Comment by beescuit** [comment]
... I think that we can merge this

**Comment by Doges** [comment]
> ... I think that we can merge this

"conflicting files"

**Comment by bolsomito** [comment]
*ha*

**Comment by beescuit** [comment]
k i forgot that I had to do it myself

**Comment by beescuit** [comment]
:HaHaa: pls review dis thanks

File: src/locales/en-US/commands.json [hunk 4]
```diff
@@ -413,6 +413,11 @@
     "tooLongText": "Your text is longer than 150 characters",
     "changedSuccessfully": "Personal text changed to {{profileText}}!"
   },
+  "guildicon": {
+    "commandDescription": "Shows current server icon",
```
[the remark this answers is not in the document]
Comment (bolsomito) [reply src/locales/en-US/commands.json]: Agreed.

[the remark this answers is not in the document]
Comment (beescuit) [reply src/locales/en-US/commands.json]: Yeah, that's better. Can you do that for me? I'm going to school rn

[the remark this answers is not in the document]
Comment (LogGame-HU-DEaN) [reply src/locales/en-US/commands.json]: Donezo.

**Comment by Doges** [comment]
s!guildicon [id] why not

**Pull request merged**